wasm/Cargo.toml

```diff
@@ -37,8 +37,3 @@ wasm-bindgen-test = "0.3.13"
 
 [profile.release]
 opt-level = "s"
-
-# We set a blank 'workspace' block disassociate this crate
-# with the workspace at the repo root. Doing so models
-# how an external developer would use it.
-[workspace]
```

Comment:
without necessarily assuming this was good to start, why don't we need it now?

Reply:
In order for our docs-generation scripting to include the `penumbra-wasm` crate, we must pull it into the top-level workspace. Otherwise:

```
error: package ID specification `penumbra-wasm` did not match any packages
```

While we could rig up some special-casing where we build the `wasm` docs separately and then edit the index HTML to include it, it doesn't seem worthwhile. We'd do better to have the docs integrated with the other crates on rustdoc, and flesh out the tests and functionality in the wasm crate.

Reply:
Right, that makes sense, what I meant was more, are we giving up anything important by moving it into the workspace?

Reply:
No, I don't think we lose much: it was always aspirational to have a way to test how the wasm bindings would be used, especially when they lived in a different repo (#1990). Now that we have the dapp site built using the wasm bindings published to npm (#2216), that's a better canary for the build pipeline, because it's actually being used in a deploy. The Zpoken folks are still maintaining the publish-to-npm workflow, but I don't see a need for us to try to host that.